src/js/profile/wearable/widget/wearable/SnapListview.js

```diff
@@ -744,7 +744,9 @@
 
 				targetIndex = getIndexOfSnapListItem(targetListItem, listItems);
 
-				utilEvent.preventDefault(e);
+				if (e.target.tagName === "LABEL") { // prevents duplicate events on input inside label;
```

Comment:
In here we need add also check if label contains input type radio. eg. e.target.querySelector("input[type='radio']")

Reply:
I think we need to more check, which is correct behavior.

Reply:
> I think we need to more check, which is correct behavior.

Yes. I agree with you.